service/src/main/java/org/cbioportal/genome_nexus/service/enricher/MyVariantInfoAnnotationEnricher.java

```diff
@@ -52,7 +52,9 @@ public void enrich(List<VariantAnnotation> annotations) {
         try {
             List<MyVariantInfo> myVariantInfo = myVariantInfoService.getMyVariantInfoByAnnotation(annotations);
             Map<String, MyVariantInfo> hgvsToMyVariantInfoMap =
-                myVariantInfo.stream().collect(Collectors.toMap(MyVariantInfo::getHgvs, v -> v));
+                myVariantInfo.stream()
+                    .filter(info -> info.getHgvs() != null)
+                    .collect(Collectors.toMap(MyVariantInfo::getHgvs, v -> v));
```

Comment:
I've added all the changes from https://github.com/genome-nexus/genome-nexus/pull/457. Signal BRCA2 gene page (https://www.signaldb.org/gene/BRCA2) is missing all clinvar, this filter could be a quick fix for now. For variants which don't have hgvs, we don't include them in hgvsToMyVariantInfoMap

Reply:
Just reiterating here ... this patch is hiding the problem. *We* constructed the hgvs query string ... so failing to get it back in the response from myvariant.info is a side-effect of the problem we have parsing the (altered) myvariant.info response schema. We should remove this patch as soon as we fix the model ... otherwise we will be blind to future degradation in myvariant.info responses as their model continues to evolve.